test/system/037-runlabel.bats

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bats
+
+load helpers
+
+@test "podman container runlabel test" {
+    tmpdir=$PODMAN_TMPDIR/runlabel-test
+    mkdir -p $tmpdir
+    containerfile=$tmpdir/Containerfile
+    cat >$containerfile <<EOF
+FROM $IMAGE
+LABEL  INSTALL  /usr/bin/podman  run  -t  -i  --rm  \\\${OPT1}  --privileged  -v /:/host --net=host --ipc=host --pid=host -e HOST=/host -e NAME=\\\${NAME} -e IMAGE=\\\${IMAGE} -e CONFDIR=/etc/\\\${NAME} -e LOGDIR=/var/log/\\\${NAME} -e DATADIR=/var/lib/\\\${NAME} \\\${IMAGE} \\\${OPT2} /bin/install.sh \\\${OPT3}
+EOF
+
+    run_podman build -t runlabel_image $tmpdir
+
+    run_podman container runlabel --opt1=OPT1 --opt2=OPT2 --opt3=OPT3 --name test1 --display  install runlabel_image
+    is "$output"   "command: bin/podman run -t -i --rm OPT1 --privileged -v /:/host --net=host --ipc=host --pid=host -e HOST=/host -e NAME=test1 -e IMAGE=localhost/runlabel_image:latest -e CONFDIR=/etc/test1 -e LOGDIR=/var/log/test1 -e DATADIR=/var/lib/test1 localhost/runlabel_image:latest OPT2 /bin/install.sh OPT3"   "generating runlabel install command"
```

Comment:
suggestion: `$PODMAN` instead of `bin/podman`. This should cover all cases including podman-remote.